conf/igenomes.config

```diff
@@ -21,7 +21,7 @@ params {
       readme      = "${params.igenomes_base}/Homo_sapiens/Ensembl/GRCh37/Annotation/README.txt"
       mito_name   = "MT"
       macs_gsize  = "2.7e9"
-      blacklist   = "${baseDir}/assets/blacklists/GRCh37-blacklist.bed"
+      blacklist   = "${projectDir}/assets/blacklists/GRCh37-blacklist.bed"
```

Comment:
Yeah, we understand - @charlotte-west and I gave this a lot of thought about whether or not to include the `smrna` in the `igenomes.config` but decided on the split approach for a few reasons:

- The smRNA are assembly agnostic (to answer your earlier question) - rRNA and customised groups of tRNAs
- It is possible, and indeed quite likely, that a user will want to use the smRNA, but not iGenomes (esp. as some of the GTFs in iGenomes are old/not compatible with some of the peak callers)
- To future-proof against a move away from iGenomes

We set up the code in `main.nf` so that for the user it is hopefully seamless - i.e. if you want to just use iGenomes, then you just specify `--genome` and it automatically does the rest. But if you want more control then you can specify e.g. `--smrna_org` (which can be e.g. `human`, `hg38` or `GRCh38` etc.) and also a human FASTA and GTF from Gencode.

Reply:
Ok. Understandable 👍🏽 Be good if we can upload these files to nf-core/test-datasets instead as I commented here https://github.com/nf-core/clipseq/pull/28#discussion_r577594688.